tailwind.config.js

```diff
@@ -1,7 +1,10 @@
-module.exports = {
-  purge: [
-    './src/**/*.vue',
-    './src/**/*.js'
+import { resolve } from 'path'
+
+export default {
+  darkMode: 'class',
```

Comment:
This allows us to [toggle dark mode manually](https://tailwindcss.com/docs/dark-mode#toggling-dark-mode-manually) with the newer version of tailwind.